starter-code/src/functions-and-arrays.js

```diff
@@ -52,6 +125,18 @@ const wordsFind = [
   'disobedience'
 ];
 
+function doesWordExist(arr, word) {
+  if(arr.includes(word)) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
You can use a short expression like return arr.includes(word), it should return true or false already by itself